data/json/monsters/zed_skeletal.json

```
@@ -71,7 +71,7 @@
     "upgrades": { "half_life": 12, "into": "mon_skeleton_hulk" },
     "death_drops": "default_zombie_clothes",
     "death_function": [ "NORMAL" ],
-    "flags": [ "SEES", "HEARS", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
+    "flags": [ "SEES", "HEARS", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
```

```suggestion
    "flags": [ "SEES", "HEARS", "HARDTOSHOOT", "REVIVES", "NO_BREATHE", "POISON", "FILTHY" ]
```
On the grounds that its armor has a more reasonable potential of being smooth, from the description, and moreover it has a lower armor vs bullets than the skeletal zombie.

---

I feel like the skeleton evolutionary line should:
A._ Be all focused around the concept of "Bones evolved in such a way that its hard to shoot"
B._ Be all focused around the concept of "Bone armor, remove hard to shoot"
Or maybe have 2 different evolutionary lines for the skeleton, each based around a different specialization idea.
Whatever the case, this kind of things should be mentioned in the description, its already very confusing as it is, and leaving only one of the skeletons with the "HARDTOSHOOT" flag can be pretty confusing for the player (Base skeleton is easy to hit, its evolution its not, and then the next evolution is easy to hit again).

---

Point.

---

so that the skeletal zombie has 6 more bullet armor than the skeletal zombie hulk?

---

Currently, the skeletal zombie (30 vs bullets) _**already**_ has 6 more bullet armor than the skeletal zombie hulk (24 vs bullets).

---

oh wait, you mean the brute, not the hulk, right? one sec

---

Whoops, sorry!

---

can I count this conversation as resolved? I agree with Termineitor244 that either the whole line should keep (and in the case of skeletal shocker get) the hardtoshoot flag or none of them should.